CourseStream/CourseStream.Data/Migrations/20231122174018_AddEnrollmentTimestamp.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace CourseStream.Data.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddEnrollmentTimestamp : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<DateTimeOffset>(
+                name: "EnrollmentTimestamp",
+                table: "EnrolledStudents",
+                type: "timestamp with time zone",
```

Comment:
It should be in UTC. Try to find how u can achieve this.

Timezones here will bring us a lot of mess and complexity to manage them

Reply:
Stored in `DateTime` format. Gathered by `var currentTime = DateTime.UtcNow;`. Gathering moved to service logic . Pls confirm if this addressed this issue. 
In db it ll this: 
(second record was made at 18:51 in Ukraine UTC: +2)
<img width="643" alt="Screenshot 2023-11-23 at 19 03 41" src="https://github.com/AcademicHubOrg/AcademicHub/assets/94055869/8ea6a69d-1076-463b-b018-ebcdb8c1ab90">